tests/llmcompressor/transformers/obcq/test_consecutive_runs.py

```diff
@@ -36,19 +41,29 @@ def _test_consecutive_runs(
             oneshot_device=self.device,
             clear_sparse_session=False,
         )
-        first_tiny_model = get_session_model()
+
+        first_model = AutoModelForCausalLM.from_pretrained(
+            self.output_first,
+            device_map="auto",
+            quantization_config=self.quantization_config,
+        )
+
         layer_0_sparse = tensor_sparsity(
-            first_tiny_model.model.layers[0].self_attn.k_proj.weight
+            first_model.model.layers[0].self_attn.k_proj.weight
         )
         assert math.isclose(layer_0_sparse.item(), 0.5, rel_tol=tolerance)
-        assert qat_active(first_tiny_model)
+        assert qat_active(first_model)
 
         session = active_session()
         session_recipe = session.lifecycle.recipe_container.compiled_recipe
         stages = [stage.group for stage in session_recipe.stages]
         self.assertEqual(len(stages), 1)
         session.reset()
 
+        recipe = infer_recipe_from_model_path(model_path=self.output_first)
```

Comment:
Can you explain why we need this first recipe?
Seems like all `initialize_recipe` does is create the session and the pre_initialize_ method uses the recipe path, but is that doing anything here?

Reply:
Yeah so we reset the session - if no reset the modifiers are finalized.
After reset, the recipe container is reset. We have a check at line 78 on this diff that checks the length of the applied recipes. 
Its not doing anthing else